src/Server/Services/Analytics/AnalyticsService.lua

```diff
@@ -504,11 +786,11 @@ end
 		value = 1
 	})
 	```
-	
+
 	@param data PlayerEvent
-	@return { [any]: any }
+	@return Promise
 ]=]
-function AnalyticsService:LogPlayerEvent(data: PlayerEvent): { [any]: any }
+function AnalyticsService:LogPlayerEvent(data: PlayerEvent)
```

Comment:
```suggestion
function AnalyticsService:LogPlayerEvent(data: PlayerEvent): {[any]: any}
```

Outside of a custom defined promise type, {[any]: any} is the closest to a return type for a promise in my opinion.